app/src/main/java/org/simple/clinic/bp/entry/ConfirmRemoveBloodPressureDialog.kt

```diff
@@ -22,6 +23,9 @@ import java.util.UUID
 import javax.inject.Inject
 
 class ConfirmRemoveBloodPressureDialog : AppCompatDialogFragment() {
+  interface RemoveBloodPressureListener {
```

Comment:
Our convention is generally to keep declared interfaces in the bottom of the class. Could we follow this convention please?

Reply:
Done.